app/model/SubscriptionData.scala

```diff
@@ -54,8 +74,8 @@ object SubscriptionData {
       lineTwo = u.privateFields.getOrBlank(_.billingAddress2),
       town = u.privateFields.getOrBlank(_.billingAddress3),
       postCode = u.privateFields.getOrBlank(_.billingPostcode),
-      countyOrState = u.privateFields.getOrBlank(_.country),
```

Comment:
Note that we were sending **country** to **county/state**. I don't know if there might be implication, but a couple of people might now see that they live in the county of "United Kingdom"